src/rhsmlib/dbus/objects/register.py

```diff
@@ -181,6 +182,33 @@ def get_organizations(self, options: dict) -> List[dict]:
         owners: List[dict] = uep.getOwnerList(options["username"])
         return owners
 
+    def get_environments(self, options: dict) -> List[dict]:
+        """Get environments for every org belonging to user
+
+        :param options: Connection options including the 'username', 'password', and 'org_id' keys
+        :return: List of environments
+        """
+        uep: UEPConnection = self.build_uep(options, basic_auth_method=True)
+        environment_service: EnvironmentService = EnvironmentService(uep)
+
+        environments = environment_service.list(options["org_id"], typed_environments=True)
+
+        keys_to_remove: List[str] = [
+            "created",
+            "updated",
+            "contentPrefix",
+            "owner",
+            "environmentContent",
```

Comment:
last nitpick of the round :) would you please alpha-sort this list, so it's easier to scan visually?